manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -217,9 +218,22 @@ func (r *FybrikApplicationReconciler) checkReadiness(applicationContext Applicat
 				setErrorCondition(applicationContext, assetID, err.Error())
 				continue
 			}
+			reqResource := dataCtx.Requirements.FlowParams.ResourceMetadata
+			if reqResource != nil {
+				// we assume to have only the geography field set at this point
+				// in the provisionedBucket ResourceMetadata
+				geo := provisionedBucketRef.ResourceMetadata.Geography
+				if reqResource.Geography != "" && geo != reqResource.Geography {
+					// log an error and make a new attempt to register the asset
```

Comment:
Who will make a new attempt? Print warning that the geography field is going to be ignored, and continue with registration.

Reply:
Fixed. Thanks